pkgs/games/dwarf-fortress/wrapper/default.nix

```diff
@@ -67,7 +68,8 @@ let
     substituteInPlace $out/data/init/init.txt \
       --replace '[INTRO:YES]' '[INTRO:${unBool enableIntro}]' \
       --replace '[TRUETYPE:YES]' '[TRUETYPE:${unBool enableTruetype}]' \
-      --replace '[FPS:NO]' '[FPS:${unBool enableFPS}]'
+      --replace '[FPS:NO]' '[FPS:${unBool enableFPS}]' \
+      --replace '[SOUND:YES]' '[SOUND:NO]'
```

Comment:
I do believe you need to call unBool here.

Reply:
Oh, how embarrassing, I squashed my test commit into the contribution. Thank you @Baughn, I've fixed it.